docs/api/reference/globals.md

```diff
@@ -0,0 +1,72 @@
+
+# Globals
+## roca(args = {})
+Roca only has one global function &mdash; `roca` &mdash; used to initialize a new Roca test instance.  You probably want to call it only once.
```

Comment:
nit: I suppose this is once per test suite? 😄

Reply:
yuuuuup, let me edit